ansible/playbooks/roles/repository/files/download-requirements/requirements/x86_64/images.yml

```diff
@@ -173,6 +173,10 @@ images:
     'calico/node:v3.23.3':
       sha1: 85a3499d4fb7e93ab6334beed57b635d09f02d1f
 
+    # for HA configuration
```

Comment:
I would move it up (e.g. to be the first one in the `kubernetes-master` group) since it's K8s version independent and rarely changed.